react/week2/src/App.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from "react";
+import "./index.css";
+import Timer from "./timer";
+class Todo extends Component {
+          constructor(props) {
+                    super(props);
+                    this.state = {
+                              list: [],
+                              todoList: this.props.todoList,
```

Comment:
Didn't need two state variables here. Also better to use the callback pattern here.